src/components/NetworkAlert/NetworkAlert.tsx

```diff
@@ -150,7 +150,12 @@ export function NetworkAlert() {
     return null
   }
 
-  const { label, logoUrl, bridge } = getChainInfo(chainId)
+  const chainInfo = getChainInfo(chainId)
+  if (!chainInfo) {
+    return null
+  }
```

Comment:
nit: I find guards more readable as one-liners. [This was Google's policy](https://google.github.io/styleguide/tsguide.html#control-flow-statements-blocks), but is not ours.

```suggestion
  if (!chainInfo) return null
```